tox.ini

```diff
@@ -7,6 +7,7 @@ basepython=
     py36: python3.6
 deps=
     dj111: Django>=1.11,<1.12
+    dj22: Django>=2.2,<2.3
```

Comment:
`dj22` should be added to the `envlist` on line 3

Reply:
Good catch. Added `dj22` to `envlist` in `tox.ini` as requested.